BUILD

```diff
@@ -143,3 +105,70 @@ py_binary(
         requirement("yarl"),
     ],
 )
+
+#pkg_files(
+#    name = "python_files",
+#    srcs = ["@python39_x86_64-unknown-linux-gnu//:files"],
+#    strip_prefix = "",
+#)
+
+#pkg_tar(
+#    name = "python_layer_tar",
+#    srcs = [":python_files"],
+#)
+
+#container_layer(
+#    name = "python_layer",
+#    directory = "/usr/local",
+#    tars = [":python_layer_tar"],
+#)
+
+#container_image(
+#    name = "python_base",
+#    base = "@amazon2_linux//image",
+#    layers = [":python_layer"],
+#)
```

Comment:
This is an alternate approach of using the rules_python interpreter in the image.

I ran into problems here because
- when passing the `:files` target directly, it would lose the directory heirarchy and flatten all the files
- when wrapping in a pkg_files it would reset the mode of all files to be non-executable